wouso/resources/templates/cpanel/proposed_home.html

```diff
@@ -0,0 +1,237 @@
+{% extends 'cpanel/index.html' %}
+{% load i18n %}
+{% load user %}
+{% load django_bootstrap_breadcrumbs %}
+
+{% block sectiontitle %}
+    Proposed Question
+
+    {% block breadcrumbs %}
+
+        {{ block.super }}
+
+        {% breadcrumb "Proposed Questions" "proposed_home" %}
+
+    {% endblock %}
+
+{% endblock %}
+
+{% block sectioncontent %}
+
+    <div class="container">
+
+        <ul class="nav nav-tabs">
+            <li><a href="#workshop">Workshop</a></li>
+            <li><a href="#quest">Quest</a></li>
+            <li><a href="#special_challenge">Special Challenge</a></li>
+            <li class="active"><a href="#qotd">Question of the Day</a></li>
+            <li><a href="#challenge">Challenge</a></li>
+            <li><a href="#quiz">Quiz</a></li>
+        </ul>
+
+
+        <div class="tab-content">
+
+            <div id="workshop" class="tab-pane fade in active">
+
+            {% for question, answers in questions_workshop.items %}
+
+                <table  class="table table-bordered table-striped">
+                    <tr><td style="text-align: left">Question:<br>{{ question.text }}</td></tr>
+                    {% for answer in answers %}
+                            <tr><td style="text-align: left{% if answer.correct %};background-color: #00FF99 {% endif %}">Answer:<br>{{ answer.text }}</td></tr>
+                        {% endfor %}
+                    <tr> <td><button type="button" class="btn btn-info" data-toggle="collapse" data-target="#td_{{ question.id }}">Details</button></td></tr>
+
+                    <tr>
+                        <td  id="td_{{ question.id }}" class="collapse" style="text-align: left">
+                            <div id="div_{{ question.id }}">
+
+                                <form action="" method="POST">
+                                    {% csrf_token %}
+                                    {{form.as_p}}
+                                    <button id="acceptBtn" type="submit" style="background-color: #00FF99" >Accept</button>
```

Comment:
@vladeugeniu do not put style on items. Use css classes. Also, the id is not really good since it is not item-unique. This should be a class